rust/actyx/ax/src/cmd/users/pubkey.rs

```diff
@@ -0,0 +1,33 @@
+use crate::{
+    cmd::{AxCliCommand, KeyPathWrapper},
+    private_key::AxPrivateKey,
+};
+use crypto::PublicKey;
+use futures::{stream::once, FutureExt, Stream};
+use structopt::StructOpt;
+use util::formats::ActyxOSResult;
+
+#[derive(StructOpt, Debug)]
+#[structopt(version = env!("AX_CLI_VERSION"))]
+pub struct PubkeyOpts {
+    #[structopt(short, long, value_name = "FILE_OR_KEY", env = "AX_IDENTITY", hide_env_values = true)]
+    /// File from which the identity (private key) for authentication is read.
```

Comment:
Question: Why add the comment after the attribute and not before?

Reply:
It makes no difference, we can settle on the other order if you want (currently it isn’t consistent).

Reply:
Personally I prefer the comment above since it "improves reading locality":
```
comment
code (attribute)
code (actual code)
```

Reply:
Okay, fixing it throughout.